src/domain/vesolje.py

```diff
@@ -7,8 +7,14 @@
 
 @dataclass
 class Vesolje:
+    st_vesoljcev: int = 8
+    razmik_vesoljcev_x: int = 0.1
+    razmik_vesoljcev_y: int = 0.1
```

Comment:
Namesto 2 spremenljivk lahko ustvariš dataclass...
```python
class Vector2D:
   x: float
   y: float
```

pa potem lahko tako narediš...
```python
razmik_vesoljcev: Vector2D = Vector2D(x=0.1, y=0.1)
```

Pa še nekaj če je to lastnost ki se tiče marsovca zakaj tega ne daš v vesoljca???